server/sonar-server/src/main/java/org/sonar/server/project/ws/BulkDeleteAction.java

```diff
@@ -67,20 +71,31 @@ public void define(WebService.NewController context) {
       .createParam(PARAM_KEYS)
       .setDescription("List of project keys to delete")
       .setExampleValue(KEY_PROJECT_EXAMPLE_001);
+
+    support.addOrganizationParam(action);
```

Comment:
it might be worse mentioning in the WS's description that the parameter is actually mandatory now, unless the user is root.

As such, backward compatibility is only kept because being admin of the default organization are also root. But I'm afraid the day we remove this hack, the compatibility will be broken and we won't notice. Therefor I suggest to keep these changes but document them mentioning root (ie. document the backward incompatible change while the implementation still is).